src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -281,6 +410,19 @@ FlutterFragment flutterFragment = FlutterFragment.withCachedEngine("my_engine_id
     .build();
 ```
 
+Kotlin:
+```java
+// With a new FlutterEngine.
+val flutterFragment = FlutterFragment.withNewEngine()
+    .renderMode(FlutterView.RenderMode.texture)
+    .build();
```

Comment:
be consistent with these semi-columns (i.e. I don't think you need them anywhere)

Reply:
Yeah, looks like a copy mistake. Removed the semicolons and changed the markdown code type from java to kotlin.